plugins/source/aws/codegen/recipes/cloudwatchlogs.go

```diff
@@ -8,6 +8,13 @@ import (
 
 func CloudWatchLogsResources() []*Resource {
 	resources := []*Resource{
+		{
+			SubService:   "resource_policies",
+			Struct:       &types.ResourcePolicy{},
+			Description:  "https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_ResourcePolicy.html",
+			PKColumns:    []string{"policy_name"},
+			ExtraColumns: defaultRegionalColumnsPK,
```

Comment:
Nit: I think using 

```
			PKColumns:    []string{"account_id", "region", "policy_name"},
			ExtraColumns: defaultRegionalColumns,
```

would make the actual PKs a bit clearer.

Reply:
fixed